modules/services/dunst.nix

```diff
@@ -64,6 +64,20 @@ in {
         description = "Package providing <command>dunst</command>.";
       };
 
+      commandOptions = mkOption {
```

Comment:
isn't it possible to achieve that via the configuration system already ? the example shows how to specify an alternative config but that's not a desirable thing IMO.

Reply:
Unfortunately, it is not. Please see my justification in the main thread.